app/controllers/orders_controller.rb

```diff
@@ -2,7 +2,7 @@ class OrdersController < ApplicationController
 
   def index
     if current_user && current_user.id == params[:user_id].to_i
-      @user = User.find(params[:user_id])
+      @user = User.find(order_params[:user_id])
       render :index
```

Comment:
What's the difference between render and redirect_to again? Is this accurate: 

redirect_to
Does a full page redirect sending a 302 html code
All variables will be lost
Sends a new request to the system

render()

Sends a 200 html code response
Holds the state of variables
The action is not executed, only the view is rendered